apps/compiler/includes/functions.sh

```diff
@@ -46,7 +46,7 @@ function comp_configure() {
 
 
 function comp_compile() {
-  [ $MTHREADS == 0 ] && MTHREADS=`grep -c ^processor /proc/cpuinfo` && MTHREADS=$(($MTHREADS + 2))
+  [ $MTHREADS == 0 ] && MTHREADS=$(grep -c ^processor /proc/cpuinfo) && MTHREADS=$(($MTHREADS + 2))
```

Comment:
not sure if are needed the double quote outside $(grep...) or in $(git...)

Reply:
?

Reply:
example
```bash
_cur_branch="$(git --git-dir="$J_PATH/.git/" --work-tree="$J_PATH/" rev-parse --abbrev-ref HEAD)"
```

or something similar

Reply:
Not according to codefactor, but I never programmed in whatever language this is